.github/pull_request_template.md

```diff
@@ -7,7 +7,8 @@ Fixes #issue.
 
 # Checklist
 
-If these changes modify code paths involving cryptography, the opening of files in VMs, network (via the RPC service) traffic, or fine tuning of the graphical user interface, Qubes testing is required. Please check as applicable:
+If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic. Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:
```

Comment:
nice change, just think there should be a `,` instead of a `.` here: `traffic. Qubes`